### ISSUE TITLE
`NaN < 0` errors

### ISSUE THREAD

**Issue by rben01** [sharkdp/numbat#644]
I would expect that `NaN < 0` be `false`, but instead it gives

```
>>> NaN < 0.0
error: runtime error
 = Conversion error: unit '' can not be converted to ''
```

**Comment by rben01** [comment]
Is this due to `f64::partial_cmp` returning `None` despite all scalars being units-compatible?

https://github.com/sharkdp/numbat/blob/f064611bcc3378685c6b9667d9f9cfe46b104f8c/numbat/src/vm.rs#L755-L759

**Comment by sharkdp** [comment]
Thank you for reporting this.

**Comment by sharkdp** [comment]
Fixed in v1.15.
